### PR TITLE
Menu Items Visibility  During Startup

### LINKED ISSUE #6045: Minor oddity with the new startup window in R-Instat
I have given it the label here reogrid as representing the data window.
There are 3 parts to the content of this new window:

a) Shortcuts to File New, Open and Library
b) Recent files
c) Help information

Now, when you start, a) and the heading for b) and c) appear immediately.  Then there is a pause - a few seconds and then the files for b) appear.

To me this looks odd.  I don't know if you agree?

If so, then could we simply swap b) and c)?  Then it will simply add the recent files to the bottom, when they appear.  And that is at the same time as some commands appear in the output window - so logical and fine?

### PR DESCRIPTION
**Pull request by Patowhiz**
Fixes #6045
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@shadrackkibet please could you review? thanks

**Comment by shadrackkibet** [comment]
This is working as expected. Only one suggestion.

- **R Packages in R-Instat** opens a dialog I think it should have an ellipsis (`...`) at the end.

File: instat/ucrDataView.vb [hunk 1]
```diff
@@ -921,11 +919,25 @@ Public Class ucrDataView
...
+    ''' toggles startup menu items visibility
+    ''' </summary>
+    ''' <param name="bVisibility"></param>
+    Public Sub StartupMenuItemsVisibility(bVisibility As Boolean)
+        panelSectionStart.Visible = bVisibility
+        panelSectionHelp.Visible = bVisibility
+        If bVisibility Then
+            HideOrShowRecentSection()
+        Else
+            panelSectionRecent.Visible = False
```
Comment (shadrackkibet) [inline instat/ucrDataView.vb:935]:
```suggestion
            panelSectionRecent.Visible = bVisibility

```

**Review by rdstern — APPROVED** [review]
Looks fine to me

**Review by dannyparsons — APPROVED** [review]
Tested and works well.

**Pull request merged**